pykarma/__init__.py

```diff
@@ -54,6 +54,10 @@ def find(url, fetch_praw=False):
         'kdtoolver': 'b1',
         'q': url
     }
+
+    if subreddit != None:
+        payload = { 'subreddit': subreddit, **payload }
```

Comment:
The `**` syntax makes this more complex than it actually is. A simple `payload['subreddit'] = subreddit` should do.

Reply:
Is it possible to specify multiple subreddits? If so you might wanna change this.